modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/am/integration/test/utils/bean/APIManageBean.java

```diff
@@ -0,0 +1,284 @@
+ *specific language governing permissions and limitations
+ *under the License.
+ */
+
+package org.wso2.am.integration.test.utils.bean;
+
+public class APIManageBean extends AbstractRequest {
+    private String name;
+    private String type = "";
+    private String version = "";
+    private String provider = "";
+    private String default_version_checked = "";
+    private String transport_https = "";
+    private String transport_http = "";
+    private String responseCache = "";
+    private String apiTier = "";
+    private String authorizationHeader = "";
+    private String backend_tps = "";
+    private String productionTps = "";
+    private String sandboxTps = "";
+    private String tier = "";
+    private String api_level_policy = "";
+    private String enableApiLevelPolicy = "";
+    private String gateways = "";
+    private String bizOwner = "";
+    private String bizOwnerMail = "";
+    private String techOwner = "";
+    private String techOwnerMail = "";
+    private String additionalProperties = "";
+    private String environments = "";
+    private String swagger = "";
+    private String tiersCollection = "";
+
+    public APIManageBean(String name, String version, String provider, String transport_https, String responseCache,
+                         String api_level_policy, String environments, String swagger, String tiersCollection) {
+        this.name = name;
+        this.version = version;
+        this.provider = provider;
+        this.transport_https = transport_https;
+        this.responseCache = responseCache;
+        this.api_level_policy = api_level_policy;
+        this.environments = environments;
+        this.swagger = swagger;
+        this.tiersCollection = tiersCollection;
+    }
+
+    @Override
+    public void init() {
+        addParameter("name", name);
+        addParameter("version", version);
+        addParameter("provider", provider);
+        addParameter("swagger", swagger);
+        addParameter("tiersCollection", "Unlimited, Gold");
+        addParameter("bizOwner", "biz_ownx");
+        addParameter("bizOwnerMail", "bbb@gmail.com");
+        addParameter("techOwner", "techowsam");
+        addParameter("techOwnerMail", "ttt@gmail.com");
+        addParameter("tiersCollection", tiersCollection);
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getType() {
+        return type;
+    }
+
+    public void setType(String type) {
+        this.type = type;
+    }
+
+    public String getVersion() {
+        return version;
+    }
+
+    public void setVersion(String version) {
+        this.version = version;
+    }
+
+    public String getProvider() {
+        return provider;
+    }
+
+    public void setProvider(String provider) {
+        this.provider = provider;
+    }
+
+    public String getDefault_version_checked() {
```

Comment:
Change getters/setters to camel case

Reply:
Fixed with cd655777f335d12a56f15e377637c8ebf0f08e7a